app/src/main/res/layout/view_user_item.xml

```diff
@@ -11,67 +11,81 @@
     android:paddingBottom="2dp"
     card_view:cardUseCompatPadding="true">
 
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="horizontal"
```

Comment:
Add `android:gravity="center_vertical"` to properly center `ivAvatar`.

Reply:
Done.